src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumber {
+    private static final String IS_NOT_LOTTO_NUMBER = "로또번호가 1~45가 아닙니다.";
+    private static final int LOTTO_MIN_NUM = 1;
+    private static final int LOTTO_MAX_NUM = 45;
+    private static final List<Integer> randomLottoNumbers = new ArrayList<>();
+
+    static {
+        for (int i = LOTTO_MIN_NUM; i <= LOTTO_MAX_NUM; i++) {
+            randomLottoNumbers.add(i);
+        }
+    }
+
+    private final int lottoNumber;
+
+    public LottoNumber(int lottoNumber) {
+        validateLottoNumbers(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    public static List<Integer> generateAutoLottoNumbers() {
+        List<Integer> numbers = new ArrayList<>(randomLottoNumbers);
+        Collections.shuffle(numbers);
+        return numbers.subList(0, 6);
+    }
```

Comment:
```suggestion
    public static List<Integer> generateAutoLottoNumbers(int requestSize) {
        List<Integer> numbers = new ArrayList<>(randomLottoNumbers);
        Collections.shuffle(numbers);
        return numbers.subList(0, requestSize);
    }
```
(반영하지 않으셔도 됩니다.) 의도했던 바는 이런식으로 요청값을 받는 것이에요. 결국 6이라는 `Lotto`가 가질 수 있는 갯수로 `Lotto`에서만 관리된다고 생각한다면 `Lotto`는 6이라는 상수를 가지고 이 메소드를 호출하도록하면 필요한 도메인 정보가 응집할 수 있겠죠 :)